terraso_backend/config/urls.py

```diff
@@ -77,4 +79,10 @@
     path("storage/", include("apps.storage.urls", namespace="terraso_storage")),
     path("shared-data/", include("apps.shared_data.urls", namespace="shared_data")),
     path("story-map/", include("apps.story_map.urls", namespace="story_map")),
+    path(
+        "sitemap.xml",
```

Comment:
Issue: This path should also explicitly state that this is related to the web client, also depending on how the rest of the code gets restrucutred this path might not be referenced directly here